toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -50,6 +50,11 @@ const (
 	// /boot directory should be only accesible by root. The directories need the execute bit as well.
 	bootDirectoryFileMode = 0400
 	bootDirectoryDirMode  = 0700
+
+	CmdlineSELinuxSettings       = "security=selinux selinux=1"
+	CmdlineSELinuxForceEnforcing = CmdlineSELinuxSettings + " enforcing=1"
+
+	SELinuxConfigFile = "etc/selinux/config"
```

Comment:
Could we add documentation comments to the public constants? Same as you did in `selinux.go`.

Reply:
fixed